### PR TITLE
(docs) Cookbook "Sorting Lists and Arrays" with the Standard Library

### PR DESCRIPTION
**Pull request by cuihtlauac**
Adapted from @F-Loyer

### CODE REVIEW

**Review by christinerose — APPROVED** [review]

**Review by christinerose — APPROVED** [review]

**Review by sabine — APPROVED** [review]
LGTM, can merge after @christinerose approves line editing.

**Review by christinerose — APPROVED** [review]

**Pull request merged**
